jaxrs-clients/src/test/java/com/palantir/remoting3/jaxrs/feignimpl/GuavaOptionalAwareDecoderTest.java

```diff
@@ -44,6 +45,9 @@
     @Rule
     public ExpectedException expectedException = ExpectedException.none();
 
+    @Rule
+    public Timeout timeout = Timeout.millis(2000);
```

Comment:
why is this rule required?

Reply:
Looks like https://github.com/palantir/http-remoting/pull/592/commits/8c1d3df0d6863fc64912686ed8715bf53f92d479 "2second timeout to make unit tests fail fast" by @iamdanfox, not sure if we still want it.

Reply:
I think we want to keep it as otherwise the way these tests fail is hanging forever. When this happens, you don't get any info about what failed on circle.